src/index.js

```diff
@@ -26,6 +48,11 @@ function removeProduct(event) {
   const target = event.currentTarget;
   console.log('The target in remove is:', target);
   //... your code goes here
+  /* const productRemoved = document.getElementsByClassName('product')
+  const removeButton = document.getElementsByClassName('btn-remove')
+  productRemoved.removeChild(removeButton) */
+  let parent = target.parentNode.parentNode
+  parent.remove()
 }
 
 // ITERATION 5
```

Comment:
For the iteration 5, we need to target the inputs and store their values, target the table, create a new element and try to pass the values from the inputs to the new created element.
Here's a solution you could've used:
```
function createProduct() {
  // Targeting all the create product inputs that the user will fill
  let newProductInputs = document.querySelectorAll(".create-product input");

  // Save the value of the inputs in variables
  let newProductName = newProductInputs[0].value;
  let newProductPrice = newProductInputs[1].value;

  // Saving the newProduct html code in a variable
  let newProduct = `
  <tr class="product">
    <td class="name">
      <span>${newProductName}</span>
    </td>
    <td class="price">$<span>${newProductPrice}</span></td>
    <td class="quantity">
      <input type="number" value="0" min="0" placeholder="Quantity" />
    </td>
    <td class="subtotal">$<span>0</span></td>
    <td class="action">
      <button class="btn btn-remove">Remove</button>
    </td>
  </tr>`;

  // Targeting the table
  const tableBody = document.querySelector("tbody");

  // Adding the newProduct to the end of the table
  tableBody.insertAdjacentHTML("beforeend", newProduct);

  // Adding the removeProduct function to the remove button
  let removeBtn = document.querySelectorAll(".btn-remove");
  removeBtn.forEach((btn) => btn.addEventListener("click", removeProduct));
}
```